src/main/java/it/gov/pagopa/reward/notification/dto/mapper/detail/RewardOrganizationExport2ExportSummaryDTOMapper.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.reward.notification.dto.mapper.detail;
+
+import it.gov.pagopa.reward.notification.dto.controller.ExportSummaryDTO;
+import it.gov.pagopa.reward.notification.model.RewardOrganizationExport;
+import org.springframework.stereotype.Service;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.util.function.Function;
+
+@Service
+public class RewardOrganizationExport2ExportSummaryDTOMapper implements Function<RewardOrganizationExport, ExportSummaryDTO>, BaseDetailMapper {
+
+    private static final DecimalFormat percentageFormatter = new DecimalFormat("0");
+    static{
+        percentageFormatter.setRoundingMode(RoundingMode.DOWN);
+    }
+
+    @Override
+    public ExportSummaryDTO apply(RewardOrganizationExport rewardOrganizationExport) {
+        return ExportSummaryDTO.builder()
+                .createDate(rewardOrganizationExport.getExportDate())
+                .totalAmount(centsToEur(rewardOrganizationExport.getRewardsExportedCents()))
+                .totalRefundedAmount(centsToEur(rewardOrganizationExport.getRewardsResultsCents()))
+                .totalRefunds(rewardOrganizationExport.getRewardNotified())
+                .successPercentage(percentageFormat(rewardOrganizationExport.getPercentageResultedOk()))
+                .build();
+    }
+
+    private String percentageFormat(Long p) {
+        return percentageFormatter.format(p/100);
+    }
```

Comment:
it could be usefull to have an Utils method to perform, in order to re-use it for each model interested

Reply:
I see it was already implemented here:
it.gov.pagopa.reward.notification.dto.mapper.RewardOrganizationExports2ExportsDTOMapper#percentageFormat

lets move into Utils and implement just once